### ISSUE TITLE
about the input GS's position and scale normalization and inverse normalization

### ISSUE THREAD

**Issue by spatialdreamer** [ChenYutongTHU/SplatFormer#19]
in your paper ,  the  GS's position are normalized: 

![Image](https://github.com/user-attachments/assets/3d3e470b-d4fb-432f-a7cf-22c412c992d9)

However in the codes , the scale parameter is normalized too, and the inverse normalization in commented in your codes which was reaplced by normalizing C2W matrix.
So I wonder:
1 why the inverse normalization of GS's position and scale is not used after feature_predict ?
2 I have my own 3D GS data, which consist of GS points and W2C camera, how can i inverse normalize the  C2W matrix or GS's position and scale? I implemented both as following, neither worked. The former lead to false rendering result, and the latter result in negative scale values in training.
    1). inverse normalize the  C2W matrix : (1) w2c -> c2w (2 ) scaler.transform(c2w[:3,3]) (3) c2w->w2c
    2).  inverse normalize GS's position and scale:
      def unnormalized_gs(self, batch_gs, scalers): 
         if  len(scalers) != len(batch_gs):
            while len(scalers) <  len(batch_gs):
                scalers.append(scalers[-1])
        batch_unnormalized_gs = []
        for gs, scaler in zip(batch_gs, scalers):
            unnormalized_gs = {}
            for key in gs:
                if key=='means': #The predicted gs may not contain means
                    unnormalized_gs['means'] = scaler.inverse_transform(gs['means'])
                elif key=='scales':
                    unnormalized_gs['scales'] = gs['scales'] - torch.log(scaler.scale_)
                else:
                    unnormalized_gs[key] = gs[key]
            batch_unnormalized_gs.append(unnormalized_gs)
        return  batch_unnormalized_gs
